js/parts/ColumnSeries.js

```diff
@@ -624,13 +683,42 @@ seriesType('column', 'line',
         };
         return series.columnMetrics;
     },
-    hasPointInX: function (point, otherSeries) {
+    /**
+     * Find the highest number of columns per category.
+     * When `series.ignoreNulls` is `"normal"`, the `maxColumnCount`
+     * is calculated before the first series.translate().
+     *
+     * @private
+     * @function Highcharts.seriesTypes.column#getMaxColumnCount
+     * @return {number}
+     */
+    getMaxColumnCount: function () {
+        var series = this;
+        var maxColumnCount = 0;
+        series.points.forEach(function (point) {
+            var columnCount = series.getColumnCount(point);
```

Comment:
Missing verification if series is  "visible".

How about null points?

Reply:
This verification is inside `getColumnCount` method.

Reply:
Indeed, I missed that.